news/vulnerabilities.xml

```diff
@@ -1594,6 +1594,53 @@ issue.
     <advisory url="/news/secadv/20170126.txt"/>
     <reported source="OSS-Fuzz project" />
   </issue>
+
+  <issue public="20160824">
+    <impact severity="Low"/>
+    <cve name="2016-2183"/>
+    <affects base="1.0.2" version="1.0.2"/>
+    <affects base="1.0.2" version="1.0.2a"/>
+    <affects base="1.0.2" version="1.0.2b"/>
+    <affects base="1.0.2" version="1.0.2c"/>
+    <affects base="1.0.2" version="1.0.2d"/>
+    <affects base="1.0.2" version="1.0.2e"/>
+    <affects base="1.0.2" version="1.0.2f"/>
+    <affects base="1.0.2" version="1.0.2g"/>
+    <affects base="1.0.2" version="1.0.2h"/>
+    <fixed base="1.0.2" version="1.0.2i" date="20160922"/>    
+    <problemtype>cryptographic weakness</problemtype>
+    <title>SWEET32 attack on Triple-DES</title>
+    <description>
+      Because DES (and triple-DES) has only a 64-bit block size,
+      birthday attacks are a real concern. With the ability to run
+      Javascript in a browser, it is possible to send enough traffic
+      to cause a collision, and then use that information to recover
+      something like a session Cookie.
+
+      Triple-DES, which shows up as “DES-CBC3” in an OpenSSL cipher
+      string, is still used on the Web, and major browsers are not yet
+      willing to completely disable it.
+
+      If you run a server, you should disable triple-DES. This is
+      generally a configuration issue. If you run an old server that
+      doesn’t support any better ciphers than DES or RC4, you should
+      upgrade.
+
+      For 1.0.2 and 1.0.1, we removed the triple-DES ciphers from the
+      “HIGH” keyword and put them into “MEDIUM.” Note that we did not
+      remove them from the “DEFAULT” keyword.
+
+      For the 1.1.0 release, we treat triple-DES just like we are
+      treating RC4. It is not compiled by default; you have to use
+      “enable-weak-ssl-ciphers” as a config option. Even when those
+      ciphers are compiled, triple-DES is only in the “MEDIUM”
+      keyword. In addition, because this is a new release, we also
+      removed it from the “DEFAULT” keyword.
```

Comment:
I am slightly confused by the "affects" settings above which only list 1.0.2 versions. The text however basically says that 1.0.2, 1.0.1 and 1.1.0 are all affected to a greater or lesser degree.

Also "1.1.0....because this is a new release"....sounds slightly odd. I realise this is the text that we wrote at the time, and it was correct then, because we are creating a new entry this is (presumably) going to be flagged up to people and cause them to ask questions. Should we update the text to be more current?

Reply:
`s/new/major/` seems like a workable change.